pkg/clustermesh/clustermesh.go

```diff
@@ -55,17 +53,17 @@ type Configuration struct {
 	// endpoints into an existing cache
 	ServiceMerger ServiceMerger
 
-	// NodeManager is the node manager to manage all discovered remote
-	// nodes
-	NodeManager nodemanager.NodeManager
-
-	nodeObserver store.Observer
+	// NodeObserver reacts to node events.
+	NodeObserver store.Observer
 
 	// RemoteIdentityWatcher provides identities that have been allocated on a
 	// remote cluster.
 	RemoteIdentityWatcher RemoteIdentityWatcher
 
 	IPCache ipcache.IPCacher
+
+	// ClusterSizeDependantInterval allows to calculate intervals based on cluster size.
+	ClusterSizeDependantInterval kvstore.ClusterSizeDependantIntervalFunc `optional:"true"`
```

Comment:
Nit: "Dependent"

Reply:
I've used *Dependant* for consistency with the [rest of the code base](https://github.com/cilium/cilium/blob/main/pkg/node/manager/cell.go#L56-L60). Given that there are quite a lot of occurrences, I would prefer not to rename them all in this PR, as it would increase the number of modified files quite significantly.